website/docs/r/cloud_provider_snapshot_backup_policy.html.markdown

```diff
@@ -73,6 +73,47 @@ resource "mongodbatlas_cloud_provider_snapshot_backup_policy" "test" {
 }
 ```
 
+## Example Usage implementing at least one Policy Item
+
+```hcl
+resource "mongodbatlas_cluster" "my_cluster" {
+  project_id   = "<PROJECT-ID>"
+  name         = "clusterTest"
+  disk_size_gb = 5
+
+  //Provider Settings "block"
+  provider_name               = "AWS"
+  provider_region_name        = "EU_CENTRAL_1"
+  provider_instance_size_name = "M10"
+  provider_backup_enabled     = true // must be enabled in order to use cloud_provider_snapshot_backup_policy resource
+  provider_disk_iops          = 100
+  provider_encrypt_ebs_volume = false
+}
+
+resource "mongodbatlas_cloud_provider_snapshot_backup_policy" "test" {
+  project_id   = mongodbatlas_cluster.my_cluster.project_id
+  cluster_name = mongodbatlas_cluster.my_cluster.name
+
+  reference_hour_of_day    = 3
+  reference_minute_of_hour = 45
+  restore_window_days      = 4
+
+
+  policies {
+    id = mongodbatlas_cluster.my_cluster.snapshot_backup_policy.0.policies.0.id
+
+    policy_item {
+      id                 = 5f0747cad187d8609a72f546
```

Comment:
This needs some explanation since it's a hard coded value.   Perhaps this would do it:

This is the id MongoDB Atlas returns for the policy item we want to keep.  Here it is hard coded because you need to either use the actual value from the Terraform state or look to map the policy item you want to keep to it's placement in the state file array that was imported in when the cluster was originally created.  For example in the state file we are using the hourly policy is the first policy in the array so it can be referred to with
mongodbatlas_cluster.my_cluster.snapshot_backup_policy.0.policies.0.policy_item.0.id instead of the hard coded value.  Summarized: to use the state file value for a policy item you need to determine the array placement # of the same frequency_type you want to keep.

Reply:
This was resolved but it's still a hard coded value that would be hard for someone to understand where it comes from.